python/paddle/fluid/layers/nn.py

```diff
@@ -7494,3 +7495,31 @@ def affine_channel(x, scale=None, bias=None, data_layout='NCHW', name=None):
         attrs={"data_layout": data_layout},
         outputs={"Out": out})
     return out
+
+
+def hash(input, hash_size, num_hash=1, name=None):
+    """
+    hash the input
+     Args:
+        input (Variable): The input variable which is a one-hot word.
+        hash_size (int): The space size for hash algorithm.
```

Comment:
what is size? is it bytes, buckets, elements?

Reply:
Thanks for reviewing, I polished the doc in [here](https://github.com/PaddlePaddle/Paddle/pull/14101)